postgres-appliance/Dockerfile

```diff
@@ -513,9 +515,19 @@ WORKDIR $PGHOME
 COPY motd /etc/
 COPY runit /etc/service/
 COPY pgq_ticker.ini $PGHOME/
+COPY scripts bootstrap /scripts/
```

Comment:
Why this one was moved from https://github.com/zalando/spilo/pull/414/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3L542?

Reply:
My bad, is a leftover from one of my tests where I was declaring USER at the bottom. I'll revert this back.